### PR TITLE
update etl, manifest, gitops

### PR DESCRIPTION
**Pull request by wangfan860**
according to the settings in qa-anvil

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# internalstaging.theanvil.io/manifest.json
## :warning: Services on branch
- dashboard
- metadata

File: internalstaging.theanvil.io/manifest.json [hunk 1]
```diff
@@ -25,13 +25,15 @@
     "spark": "quay.io/cdis/gen3-spark:2020.06",
     "tube": "quay.io/cdis/tube:2020.06",
     "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.06",
-    "wts": "quay.io/cdis/workspace-token-service:2020.06"
+    "wts": "quay.io/cdis/workspace-token-service:2020.06",
+    "dashboard": "quay.io/cdis/gen3-statics:master",
```
Comment (mfshao) [inline internalstaging.theanvil.io/manifest.json:29]:
```suggestion
    "dashboard": "quay.io/cdis/gen3-statics:2020.06",
```

**Comment by PlanXCyborg** [comment]
# internalstaging.theanvil.io/manifest.json
## :warning: Services on branch
- dashboard

File: internalstaging.theanvil.io/manifest.json [hunk 2]
```diff
@@ -277,15 +279,15 @@
     "indices": [
       {
         "index": "etl_internalstaging-anvil",
```
Comment (mfshao) [inline internalstaging.theanvil.io/manifest.json:281]:
```suggestion
        "index": "internalstaging-anvil_etl",
```

File: internalstaging.theanvil.io/manifest.json [hunk 3]
```diff
@@ -277,15 +279,15 @@
...
-        "type": "case"
+        "type": "subject"
       },
       {
         "index": "file_internalstaging-anvil",
```
Comment (mfshao) [inline internalstaging.theanvil.io/manifest.json:285]:
```suggestion
        "index": "internalstaging-anvil_file",
```

**Pull request merged**